Automattic/sensei
# Lesson actions block

Fixes #4444

### Changes proposed in this Pull Request

* It adds a new Lesson Actions block for the Course Theme project, and it includes the old Complete Lesson and Take Quiz blocks.
* Notice that the button styles aren't good yet (theme conflicts). We need to revisit that when we update the Course Theme approach.

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Create a course with lessons (some lessons with prerequisites) and quizzes (some quizzes with pass required).
* Complete some quizzes
* Select the "Sensei Theme" as the "Course Theme" (sidebar).
* Make sure that lessons with prerequisites not completed have the actions disabled.
* Make sure that lessons with pass required don't show the complete button.
* Make sure that lessons with pass not required show both buttons.
* Make sure completed lessons don't show the buttons.
* Make sure not enrolled users don't see the buttons.
* And please, try to reproduce other weird scenarios that you can think of to make sure it's working properly for all possible scenarios.
